### DISCUSSION TITLE
Unable to stop physics body passing through another physics body

### DISCUSSION THREAD

**Discussion by webciter** [enable3d/enable3d#191] [Help / Questions] [answered]
I'v created a scene with a floor and walls on each side, but when i fire a physics body with a size of less than 1. The physics engine does not stop the item from passing through.

I'v tried editing the settings of setCcdMotionThreshold and setCcdSweptSphereRadius but seems to make things worse.

**Comment by yandeu** [comment, marked as answer]
Try increasing the 
- `subSteps to 4` and 
- `timeStep to 1 / 120`.

See https://enable3d.io/docs.html#physics-configuration

---

Also try to make the walls thicker.

**Reply by webciter** [reply]
the settings

    subSteps to 4 and
    timeStep to 1 / 120.

was not enough


    subSteps to 4 and
    timeStep to 1 / 240

worked ok.

**Reply by yandeu** [reply]
Guess the object is moving pretty fast then?

Have you tried increasing the thinkness? Should be less performance intensive.
